easybuild/tools/repository/svnrepo.py

```diff
@@ -170,3 +170,19 @@ def cleanup(self):
             rmtree2(self.wc)
         except OSError, err:
             self.log.exception("Can't remove working copy %s: %s" % (self.wc, err))
+
+    def export(self, to_path, revision=None):
```

Comment:
we need an `export` that boils down to `raise NotImplemented` in `Repository` too?